conversation/src/main/java/com/ibm/watson/developer_cloud/conversation/v1/model/WorkspaceExportResponse.java

```diff
@@ -0,0 +1,268 @@
+/*
+ * Copyright 2017 IBM Corp. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package com.ibm.watson.developer_cloud.conversation.v1.model;
+
+import java.util.List;
+import java.util.Map;
+
+import com.google.gson.annotations.SerializedName;
+import com.ibm.watson.developer_cloud.service.model.GenericModel;
+
+/**
+ * WorkspaceExportResponse.
+ */
+public class WorkspaceExportResponse extends GenericModel {
+
+  /** The name of the workspace. */
+  private String name;
+  /** The description of the workspace. */
+  private String description;
+  /** The language of the workspace. */
+  private String language;
+  /** Any metadata that is required by the workspace. */
+  private Map<String,Object> metadata;
+  /** The timestamp for creation of the workspace. */
+  private String created;
+  /** The timestamp for the last update to the workspace. */
+  private String updated;
+  /** The workspace ID. */
+  @SerializedName("workspace_id")
+  private String workspaceId;
+  /** The current status of the workspace (`Non Existent`, `Training`, `Failed`, `Available`, or `Unavailable`). */
+  private String status;
```

Comment:
This should be an Enumeration. I think that will be hard.. Is there a way to describe enumerations in swagger?